### DISCUSSION TITLE
Java does not complete code based on comments

### DISCUSSION THREAD

**Discussion by a915944224** [community/community#17486] [Copilot Conversations]
![1653882997(1)](https://user-images.githubusercontent.com/103731063/170914494-3952aa1d-a48a-4548-9bba-2f61c45690fd.jpg)

**Comment by davecheney** [comment]
Which editor and version of the copilot plugin are you using?

**Reply by a915944224** [reply]
The idea of 2021.2 Copilot 1.1.23.1570

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by a915944224** [reply]
Normal log-in authorization. That's not what caused it, is it?

**Reply by a915944224** [reply]
After uninstalling the plugin and reinstalling it, GitHub Copilot will appear when you log in: Failed to initiate the GitHub login process. Please try again
